mailpoet/lib/Cron/Workers/KeyCheck/PremiumKeyCheck.php

```diff
@@ -25,6 +25,9 @@ public function checkProcessingRequirements() {
   }
 
   public function checkKey() {
+    // for phpstan because we set bridge property in the init function
+    if (!$this->bridge) return;
```

Comment:
What happens when someone forgets to call the init function? I mean wouldn't it be safer to throw an exception? I think that with this fix we actually allowed using the method when the class is not properly initialized.

Reply:
The exception was my second idea for this. I haven't changed the behavior, but I have wanted to add the annotation for the bridge property. I can use an exception here if you think that it is better.